python/setup.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
```

Comment:
Is this still needed if we go the route of asking people to install horovod and pytorch prior to installing SynapseMl so that this can be done in regular packaging workflow?